symfony/plugins/orangehrmMaintenancePlugin/DownloadFormats/DownloadFormat.php

```diff
@@ -32,21 +33,35 @@ abstract class DownloadFormat
      */
     abstract public function getFormattedString($values);
 
+    /**
+     * @param $fileName
+     * @return string
+     */
+    public function removeNonAsciiCharacters($fileName): string
```

Comment:
example?

Reply:
test£$&*().json  replaced as test$&*().json

Reply:
I found in HeaderUtil.php inbuilt class
 ```
 // filenameFallback is not ASCII.
        if (!preg_match('/^[\x20-\x7e]*$/', $filenameFallback)) {
            throw new \InvalidArgumentException('The filename fallback must only contain ASCII characters.');
        }
```

Reply:
if the filenameFallback is empty then it's throwing an error?
but just add string value to filenameFallback then it's not throw error but I was not sure it's the best approach